AuroraAssetEditor/MainWindow.xaml

```diff
@@ -47,12 +51,25 @@
                 <MenuItem x:Name="EditMenu"
                           Header="Edit"
                           SubmenuOpened="EditMenuOpened" />
+                <MenuItem x:Name="GameSelector"
+                  Visibility="Hidden"
+                  Margin="100,0,0,0">
```

Comment:
Why the high margin?

Reply:
It is so the current game appears more in the center